docs/memfault/AWS.rst

```diff
@@ -0,0 +1,79 @@
+.. _aws-memfault-integration:
+
+Memfault Integration for AWS
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. include:: ./Index.rst
+   :start-after: intro_start
+   :end-before: intro_end
+
+The integration can be used standalone and does not required an exisiting instance of :ref:`nRF Asset Tracker for AWS <aws-getting-started>`.
+
+Prerequisites
+*************
+
+Before you start setting up the Memfault Integration for AWS, make sure that you fulfill the following prerequisites:
+
+.. include:: ../aws/GettingStarted/Index.rst
+   :start-after: prerequisites_start
+   :end-before: prerequisites_end
+
+Clone the project and install the dependencies
+**********************************************
+
+To clone the `nRF Asset Tracker Memfault integration for AWS IoT <https://github.com/NordicSemiconductor/asset-tracker-cloud-memfault-aws-js>`_ project and install the dependencies, use the following commands:
+
+.. parsed-literal::
+
+    git clone --branch |version| --single-branch \\
+      https://github.com/NordicSemiconductor/asset-tracker-cloud-memfault-aws-js.git memfault-aws
+    
+    cd memfault-aws 
+    
+    # Install the dependencies
+    npm ci
+    
+Configure Memfault settings
+***************************
+
+You can retrieve the project settings from the settings page of the Memfault
+dashboard of your organization.
+
+.. parsed-literal::
+   :class: highlight
+
+   aws ssm put-parameter --type String --name /${STACK_NAME:-nrf-memfault}/thirdParty/memfault/projectKey --value *<your memfault project key>*
+   aws ssm put-parameter --type String --name /${STACK_NAME:-nrf-memfault}/thirdParty/memfault/organization --value *<your organization slug>*
+   aws ssm put-parameter --type String --name /${STACK_NAME:-nrf-memfault}/thirdParty/memfault/project --value *<your project slug>*
+
+The organization auth token can be accessed and managed by Administrators at
+Admin → Organization Auth Tokens in the Memfault UI.
+
+.. parsed-literal::
+   :class: highlight
+
+   aws ssm put-parameter --type String --name /${STACK_NAME:-nrf-memfault}/thirdParty/memfault/authToken --value *<your auth token>*
+
+Configure stack settings
+************************
+
+The topic devices use to publish memfault chunks can be configured through the
```

Comment:
Should memfault be capitalized? Is there a difference in meaning between capitalized and non-capitalized?

Reply:
> Should memfault be capitalized? Is there a difference in meaning between capitalized and non-capitalized?

Addressed in my suggestion below.